config/locales/mailers/en.yml

```diff
@@ -16,7 +16,7 @@ en:
       account_deleted: "%{community} - Account Deletion Confirmation"
       deletion_requested: "%{community} - Account Deletion Requested"
       trusted: Congrats! You're now a "trusted" user on %{community}!
-      moderator: 'Congrats! You''re the moderator for #%{tag_name}'
+      moderator: 'Congrats! You''re now a moderator for #%{tag_name}'
```

Comment:
I have a question since I see the double `''` on this line. Will we still have the `'` in the text?